### ISSUE TITLE
Systemd service not enabled on Debian

### ISSUE THREAD

**Issue by deric** [influxdata/telegraf#525]
Debian does not contain `/etc/lsb-release` file (unlike Ubuntu), instead there's a binary `lsb_release`:
```
$ lsb_release -r
Release:        8.2
```
When release number is `>=8.0` systemd service should be enabled. Maybe just removing [this line](https://github.com/influxdata/telegraf/blob/master/scripts/post-install.sh#L59) would solve it. Assuming that most distributions would use either `systemd` or `sysvinit`.

**Comment by rossmcdonald** [comment]
@deric Thanks for letting us know. Fix here: #526 

Workaround:

```
$ sudo cp /usr/lib/telegraf/scripts/telegraf.service /lib/systemd/system/
$ sudo systemctl enable telegraf
```

**Comment by deric** [comment]
No, problem. Yeah, I've managed to bypass the issue, but it's annoying to do this on each server. Using `/etc/debian_version` is even better idea, than mine :+1:.
